dfusion_rust_core/src/models/state.rs

```diff
@@ -1,10 +1,14 @@
 use byteorder::{BigEndian, WriteBytesExt};
+use graph::data::store::{Entity, Value};
 use serde_derive::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
-use web3::types::{H256, U256};
+use std::sync::Arc;
+use web3::types::{H256, U256, Log};
 
 use crate::models::{TOKENS, RollingHashable};
 
+use super::util::{pop_u8_from_log_data, pop_u16_from_log_data, pop_h256_from_log_data, to_value};
+
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct State {
```

Comment:
Maybe its a little too late, but would it make more sense to call this AccountState?

Reply:
Yes, will make this a separate PR though.